test-case/check-sof-logger.sh

```diff
@@ -161,6 +159,14 @@ run_loggers()
              > "$data_file" 2> "$error_file" & dmaPID=$!
     fi
 
+    sleep "$logger_settle_time_secs"
+    # Avoid racing with runtime-pm by opening the PCM nodes when
+    # the loggers are running.
+    ping_pcm_devices
+
+    dlogi "Collecting logs now, last kernel activity:"
```

Comment:
```suggestion
    dlogi "Last kernel activity:"
```

I find the "Collecting logs NOW,..." message misleading for two reasons:
- This code is trying to work around the fact that we don't know _when exactly_ log collection starts.
- Seeing `journalctl` called immediately after "collecting now" gives the wrong impression that they're related to each other (I think they're not)

Reply:
Ack, will fix.